src/main/java/com/three_tech_solutions/slot_app/services/interfaces/StudentService.java

```diff
@@ -9,9 +11,12 @@
 public interface StudentService {
 
     StudentResponse createStudent(CreateStudentRequest studentDTO);
+
     void deleteStudent(UUID studentId);
 
     StudentDetailsResponse getStudentById(UUID studentId);
 
     void activateStudent(UUID studentId);
+
+    StudentResponse updateStudent (UUID studentId, UpdateStudentRequest studentUpdated);
```

Comment:
Te voy a sacar el espacio de tu compu ah
```suggestion
    StudentResponse updateStudent(UUID studentId, UpdateStudentRequest studentUpdated);
```